packages/template/webpack-typescript/package.json

```diff
@@ -29,7 +29,9 @@
     "chai": "^4.3.3",
     "fast-glob": "^3.2.7",
     "fork-ts-checker-webpack-plugin": "^7.2.13",
-    "listr2": "^7.0.2"
+    "listr2": "^7.0.2",
+    "typescript": "^4.6.3",
+    "webpack": "^5.69.1"
```

Comment:
webpack is provided via the forge plugin itself, it should not be a direct dependency of the top level templated project

Reply:
`fork-ts-checker-webpack-plugin` is a dependency of `webpack` (it have `webpack` as peer dependency). So the `fork-ts-checker-webpack-plugin` should be replaced with another dependency or handled internally in forge.